packages/lesswrong/components/review/FrontpageReviewWidget.tsx

```diff
@@ -0,0 +1,329 @@
+        <li>Any post from before 2021 can be nominated</li>
+        <li>Any user registered before the start of the review can nominate posts</li>
+        <li>Posts with at least one vote proceed to the Review Phase.</li>
+      </ul>
+      <div>If you've been actively reading {siteNameWithArticleSetting.get()} before now, but didn't register an account, reach out to us on intercom.</div>
+    </div> :
+    <div>
+      <div>Cast initial votes for the {REVIEW_YEAR} Review
+        Nominate posts for the {REVIEW_YEAR} Review</div>
+      <ul>
+        <li>Nominate a post by casting a <em>rough vote</em>, or vote on an existing nomination to help us prioritize it during the Review Phase.</li>
+        <li>Any post from {REVIEW_YEAR} can be nominated</li>
+        <li>Any user registered before {REVIEW_YEAR} can nominate posts for review</li>
+      </ul>
+    </div>
+
+  const reviewTooltip = isEAForum ?
+    <>
+      <div>Review posts for the EA Writing Mega Review (Opens {nominationEndDate.clone().add(1, 'day').format('MMM Do')})</div>
+      <ul>
+        <li>Write reviews of posts nominated for the EA Writing Mega Review</li>
+        <li>Only posts with at least one review are eligible for the final vote</li>
+      </ul>
+    </> :
+    <>
+      <div>Review posts for the {REVIEW_YEAR} Review (Opens {nominationEndDate.clone().add(1, 'day').format('MMM Do')})</div>
+      <ul>
+        <li>Write reviews of posts nominated for the {REVIEW_YEAR} Review</li>
+        <li>Only posts with at least one review are eligible for the final vote</li>
+      </ul>
+    </>
+
+  const voteTooltip = isEAForum ?
+    <>
+      <div>Cast your final votes for the EA Writing Mega Review. (Opens {reviewEndDate.clone().add(1, 'day').format('MMM Do')})</div>
+      <ul>
+        <li>Look over nominated posts and vote on them</li>
+        <li>Any user registered before {nominationStartDate.format('MMM Do')} can vote in the review</li>
+      </ul>
+      <div> Before the vote starts, you can try out the vote process on posts nominated and reviewed in {REVIEW_YEAR-1}</div>
+    </> :
+    <>
+      <div>Cast your final votes for the {REVIEW_YEAR} Review. (Opens {reviewEndDate.clone().add(1, 'day').format('MMM Do')})</div>
+      <ul>
+        <li>Look over {/* TODO: Raymond Arnold look here, sentence fragment */} </li>
+        <li>Any user registered before {REVIEW_YEAR} can vote in the review</li>
+        {!isEAForum && <li>The end result will be compiled into a canonical sequence and best-of {REVIEW_YEAR} book</li>}
+      </ul>
+      <div> Before the vote starts, you can try out the vote process on posts nominated and reviewed in {REVIEW_YEAR-1}</div>
+    </>
+
+  const dateFraction = (fractionDate: moment.Moment, startDate: moment.Moment, endDate: moment.Moment) => {
+    if (fractionDate < startDate) return 0
+    return ((fractionDate.unix() - startDate.unix())/(endDate.unix() - startDate.unix())*100).toFixed(2)
+  }
+
+  const allEligiblePostsUrl = 
```

Comment:
Dumb question: If we're calling this all 2020 posts, why filter for unnominated?

Reply:
It should maybe have a different name. Filter for unnominated is so people can focus on adding new posts.